packages/Webkul/Ui/src/Resources/views/datagrid/table.blade.php

```diff
@@ -397,23 +398,26 @@
                     formURL: function(column, condition, response, label) {
                         var obj = {};
 
-                        if (column == "" || condition == "" || response == "" || column == null || condition == null || response == null) {
+                        if (column === "" || condition === "" || response === ""
+                            || column === null || condition === null || response === null)
+                        {
                             alert('{{ __('ui::app.datagrid.filter-fields-missing') }}');
 
                             return false;
                         } else {
+
                             if (this.filters.length > 0) {
-                                if (column != "sort" && column != "search") {
-                                    filterRepeated = 0;
+                                if (column !== "sort" && column !== "search") {
+                                    let filterRepeated = false;
 
-                                    for(j = 0; j < this.filters.length; j++) {
-                                        if (this.filters[j].column == column) {
-                                            if (this.filters[j].cond == condition && this.filters[j].val == response) {
-                                                filterRepeated = 1;
+                                    for(let j = 0; j < this.filters.length; j++) {
```

Comment:
Please add space after for

Reply:
Done. 

And found some more on my way through the file.
Thanks for reviewing :)